wgsl/index.bs

```diff
@@ -6656,23 +6656,36 @@ first a [=read access=] gets the old value, and then a [=write access=] stores t
   </xmp>
 </div>
 
-Note: A compound assignment can rewritten as different WGSL code that uses a [=simple assignment=] instead.
+<div class=note><span class=marker>Note:</span>
```

Comment:
This part is a drive-by editorial polish.

Make it one div so the three paragraphs are seen as one big note.
And break the code fragments out to their own blockquotes.